common/lib/capa/capa/responsetypes.py

```diff
@@ -248,7 +248,9 @@ def render_html(self, renderer, response_msg=''):
           - response_msg: a message displayed at the end of the Response
         """
         # render ourself as a <span> + our content
-        tree = etree.Element('span')
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
```

Comment:
@ehteshamkafeel It is very helpful to a screen reader user if container elements, like section, that contain discrete blocks of related content, have an accessible name.  As @clrux said, you can do this with aria-label, or more preferably aria-labelledby(if the text that you want to use as a label is already present on the page, like when the section contains a heading that summarizes it).  However, since we have no idea how long the "question" text might be, I am a little hesitant to use that value as the accessible name.  If the question text is short, like "Which of the following is a fruit?" Than that is an acceptable length to be used as an accessible name,  But if the question is a few sentences long, it could actually create a very wordy title for the region.  Early on, I had proposed generic text like "Question 1" and "Question 2" to identify these regions (each needs to be unique, as the usefulness disappears if they are all the same.)  I was told that we wouldn't be able to keep count of the individual XBlocks on any given page since they are not aware of each other.  Do you think we would be able to do this?  I'm also tempted to use the value of the "Display name" here.  That might actually be the best solution.  We could encourage authors to make these unique, and concise.

Sorry for the very long answer to your question...